app/lib/features/space/pages/members_page.dart

```diff
@@ -72,7 +98,7 @@ class SpaceMembersPage extends ConsumerWidget {
               ),
               itemBuilder: (context, index) => MemberListEntry(
                 memberId: members[index],
-                roomId: spaceIdOrAlias,
+                roomId: widget.spaceIdOrAlias,
               ),
             );
           },
```

Comment:
add a `skipLoadingOnReload: false` to avoid that ugly flickering of the list on typing.